docs/prepare-a-shared-rp-development-environment.md

```diff
@@ -472,6 +472,16 @@ each of the bash functions below.
    when running the `deploy_env_dev_override` command, delete the `-pip` resource
    and re-run.
 
+1. Get the AKS kubeconfig and upload it to the storage account:
```

Comment:
@yehornaumenko can you also update the documentation to state that these changes should be done with `SECRET_SA_ACCOUNT_NAME=e2earosecrets`? I cannot find such information present.

Reply:
Done here https://github.com/Azure/ARO-RP/pull/2559/files#diff-9b3780a23507353cbb9a0ef8435545f8c4c6d483c105190658d2b7e60a7d7a7eR30